app-engine/java/ultimate/build.gradle

```diff
@@ -19,7 +19,7 @@ sourceSets.main.java.srcDirs = ['src']
 sourceSets.test.java.srcDirs = ['testSrc', 'testResources']
 
 intellij {
-    type 'IU'
+    type = 'IU'
```

Comment:
nit, but the `=` isn't required here right?

Reply:
It doesn't, I just wanted to unify it with the main build file where we have `type = ideaEdition` so it looks similar. I can revert this if you feel this is less intuitive.

Reply:
nah. I'm ok with it as is